daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2643,6 +2644,15 @@ func NewDeviceSet(root string, doInit bool, options []string, uidMaps, gidMaps [
 		minFreeSpacePercent:   defaultMinFreeSpacePercent,
 	}
 
+	if driverVersion, err := devicemapper.GetDriverVersion(); err == nil {
```

Comment:
There's another check here; can we reuse that option? https://github.com/moby/moby/blob/bccebdac1806d39848c6b51e9fea757e77bc2dad/daemon/graphdriver/devmapper/deviceset.go#L1521-L1524